tests/components/deconz/test_config_flow.py

```diff
@@ -330,9 +320,60 @@
     assert result["data"] == {
         config_flow.CONF_HOST: "1.2.3.4",
         config_flow.CONF_PORT: 80,
-        config_flow.CONF_API_KEY: "1234567890ABCDEF",
+        config_flow.CONF_API_KEY: API_KEY,
+    }
+
+
+async def test_create_entry_updates_configuration(hass, aioclient_mock):
+    """Test that _create_entry work and that bridgeid can be requested."""
+    gateway = await setup_deconz_integration(hass)
+
+    aioclient_mock.get(
+        f"http://2.3.4.5:80/api/{API_KEY}/config",
+        json={"bridgeid": BRIDGEID},
+        headers={"content-type": "application/json"},
+    )
+
+    flow = config_flow.DeconzFlowHandler()
+    flow.context = {}
+    flow.hass = hass
+    flow.deconz_config = {
+        config_flow.CONF_HOST: "2.3.4.5",
+        config_flow.CONF_PORT: 80,
+        config_flow.CONF_API_KEY: API_KEY,
     }
 
+    result = await flow._create_entry()
```

Comment:
Can we use the core interface instead, `hass.config_entries.flow.async_init`?

Reply:
Absolutely, Ill check it out!

Reply:
Worked through the whole suite